check_if_image_tag_exists/build.sh

```diff
@@ -2,13 +2,13 @@
 
 set -e
 
-IMAGE=$1
+export IMAGE=$1
 
 if [ -z "$1" ]; then
   echo "Usage: ./build.sh [image_path] [auth_file_remote_path]"
   echo "Please provide fully qualified path to target image."
   exit 1
 fi
 
-sed -i "s|%IMAGE%|$IMAGE|g" cloudbuild.yaml
+envsubst '${IMAGE}' < cloudbuild.yaml.in > cloudbuild.yaml
```

Comment:
I think you want to just remote the first argument here. envsubst will replace all env variables by default, this is only to make a whitelist which isn't very important here.

It also trips up shellcheck because it looks like you're trying to expand a variable, which won't work in single quotes.